css/main.css

```diff
@@ -194,8 +194,14 @@ li {
   }
 }
 
-.callout {
-  background: #f6f9fc;
+.callout-blue {
```

Comment:
This is totally just a nitpick, but I would change these classes to `callout`, `callout--blue` and `callout--green` to stick with our BEM naming. You could then share the margin and padding styles in the main `callout` class and have the modifier ones only change `background-color`.